R/sidebar.R

```diff
@@ -293,3 +279,10 @@ sidebar_dependency <- function() {
     script = "sidebar.min.js"
   )
 }
+
+sidebar_init_js <- function() {
```

Comment:
```suggestion
# Note: if we want to avoid inline `<script>` tags in the future for initialization code, 
# we might be able to do so by turning the sidebar layout container into a web component
sidebar_init_js <- function() {
```

Reply:
Note applied in 0cb6d58cf46008e99a4b2c0be1792ba748d46bcf